selfdrive/modeld/dmonitoringmodeld.cc

```diff
@@ -84,11 +83,9 @@ int main(int argc, char **argv) {
       LOGD("dmonitoring process: %.2fms, from last %.2fms", t2-t1, t1-last);
       last = t1;
     }
-
+    visionstream_destroy(&stream);
   }
 
-  visionstream_destroy(&stream);
```

Comment:
if do_exit is true while reconnecting, visionstream_destroy will be called twice here